docs/CONTRIBUTING-id.md

```diff
@@ -16,60 +16,60 @@ Dengan berkontribusi, Anda setuju untuk menghormati [Kode Etik](CODE_OF_CONDUCT-
+    - Jika tautan/konten-nya merujuk ke edisi yang berbeda, dan Anda merasa edisi tersebut cukup berbeda sehingga layak untuk tetap didaftarkan, tambahkan tautan terpisah dengan menambahkan keterangan untuk masing-masing edisi (lihat [Issue #2353](https://github.com/EbookFoundation/free-programming-books/issues/2353) untuk berpartisipasi dalam diskusi tentang pemformatan).
+- Utamakan komit atomik (satu komit per-penambahan/penghapusan/modifikasi) daripada komit yang lebih besar. Tidak perlu menggabungkan komit-komit Anda sebelum mengirimkan PR. (Kami tidak memaksakan aturan ini untuk diikuti, karena ini hanya masalah kenyamanan bagi para pengelola).
+- Jika buku/konten yang ingin didaftarkan atau terbitan lama, sertakan tanggal publikasi setelah judul dari konten/buku yang diusulkan.
+- Sertakan nama atau nama-nama penulis (jika penulis lebih dari satu). Anda dapat menyingkat daftar penulis dengan "`et al.`".
 - Jika buku belum selesai, dan masih dalam tahap pengerjaan, tambahkan keterangan "`dalam proses`" seperti yang dijelaskan [di bawah ini](#in_process).
-- Jika suatu sumber dipulihkan menggunakan [*Internet Archive's Wayback Machine*](https://web.archive.org) (atau serupa), tambahkan keterangan "`terarsip`" seperti yang dijelaskan [di bawah](#archived). Versi terbaik untuk digunakan adalah versi terbaru dan lengkap.
-- Jika alamat email atau pengaturan akun diperlukan sebelum pengunduhan, tambahkan catatan sesuai dengan bahasa yang tepat dalam tanda kurung, misalnya: `(alamat email *diminta*, tidak wajib)`.
+- Jika suatu sumber merupakan sumber yang dipulihkan menggunakan [*Internet Archive's Wayback Machine*](https://web.archive.org) (atau serupa), mohon tambahkan keterangan "`terarsip`" seperti yang dijelaskan [di bawah](#archived). Akan tetapi mohon gunakan versi terbaru dan lengkap.
+- Jika suatu sumber membutuhkan alamat email pengunduh/pengunjung atau membutukan proses pembuatan akun maka tambahkan catatan sesuai dengan bahasa yang tepat dalam tanda kurung, misalnya: `(alamat email *wajib*, tidak wajib)`.
 
 
 <!----><a id="formatting"></a>
 ### Pemformatan
 
-- Semua daftar adalah file `.md`. Coba pelajari sintaks [Markdown](https://guides.github.com/features/mastering-markdown/). Itu mudah!
-- Semua daftar dimulai dengan Indeks. Idenya adalah untuk membuat daftar dan menautkan semua bagian dan subbagian di sana. Simpan dalam urutan abjad.
-- Bagian menggunakan heading level 3 (`###`), dan subbagian menggunakan heading level 4 (`####`).
+- Semua daftar tautan ditulis pada berkas `.md`. Coba pelajari sintaks [Markdown](https://guides.github.com/features/mastering-markdown/). Itu mudah!
+- Semua daftar tautan dimulai dengan Indeks. Idenya adalah untuk membuat daftar dan menautkan semua bagian dan subbagian di sana. Simpan dalam urutan abjad alfabetis.
+- Daftar tautan menggunakan heading level 3 (`###`), dan sub-nya menggunakan heading level 4 (`####`).
```

Comment:
are u sure it is sense? I mean as Indonesian I do understand the what the sentence mean, but I don't find it is make sense, we don't says just 'bagian', it is must be followed by the context of the word, I mean even we re-paraphrase the sentences, it still give the same meaning. wdyt?

Reply:
Please take a look at this screenshot from https://github.com/EbookFoundation/free-programming-books/blob/main/books/free-programming-books-id.md#html-and-css

![image](https://github.com/EbookFoundation/free-programming-books/assets/63894003/fbb779a3-cb68-4215-9bf5-b8f5f170cbc6)

- _Section_ (Bagian) is the **green** one.
- _Subsection_ (Subbagian) is the **yellow** one.

I'm afraid people will assume the "Daftar tautan" term is the **red** one, and use `###` for the list instead of `*`.

Reply:
Are u will assume like that?

Reply:
will try to re-paraphrase that